app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/debugmenu/DebugMenuViewModel.kt

```diff
@@ -103,10 +108,15 @@ sealed class DebugMenuAction {
     data object NavigateBack : DebugMenuAction()
 
     /**
-     * The user has clicked "reset" button.
+     * The user has clicked "reset" button. For the feature flag section.
```

Comment:
Should this be `button for` instead of having the period

Reply:
yeah that would probably be a more cogent sentence 😆